mplc/docs/documentation.md

```diff
@@ -403,7 +338,7 @@ dataset = dataset.Dataset(
 ```
 
 ### Model generator
-This function provides the model which will be trained by the scenario object. Currently the library handles compiled Keras' model (see MNIST, ESC50, IMDB and CIFAR10 datasets), and Scikit-Learn Linear Regression (see the TITANIC dataset).  
+This function provides the model use, which will be trained by the scenario object. 
```

Comment:
This is weird, it looks like `documentation.md` goes back in time versus its current state in `master`?

Reply:
I think this PR will need to be rebased as you pointed before @bowni